docker/ruby/Dockerfile

```diff
@@ -35,7 +35,6 @@ RUN groupadd -g $GID $USER && \
 ENV LANG C.UTF-8
 
 # Point Bundler at /gems. This will cause Bundler to re-use gems that have already been installed on the gems volume
-ENV BUNDLE_PATH /gems
 ENV BUNDLE_HOME /gems
```

Comment:
BUNDLE_HOME is no longer a Bundler config environment variable. This could be cleaned up now too.